docs/guides/e2e-test-tips.md

```diff
@@ -2,11 +2,6 @@
 
 E2E tests are less stable and slower than unit tests therefore it's useful to know some tricks
 
-## Mac M1 and ARM support
-
-Currently, we don't support ARM architecture when building docker images which we rely on for e2e tests.
-This is tracked in [#3252](https://github.com/kumahq/kuma/issues/3252) and [#237](https://github.com/kumahq/kuma/issues/237).
-
```

Comment:
I did not go too deep into e2e tests, but both issues are closed. Can this section be removed?

Reply:
Yes, arm works fine now